app/src/main/java/org/thoughtcrime/securesms/home/ConversationOptionsBottomSheet.kt

```diff
@@ -52,6 +55,7 @@ class ConversationOptionsBottomSheet(private val parentContext: Context) : Botto
             binding.blockTextView -> onBlockTapped?.invoke()
             binding.unblockTextView -> onUnblockTapped?.invoke()
             binding.deleteTextView -> onDeleteTapped?.invoke()
+            binding.leaveTextView -> onDeleteTapped?.invoke()
```

Comment:
The leave option was already handled by the delete textview. If you look at the logic further down it was changing its text to 'Delete', 'Clear' or 'Leave' based on some criteria.
The new leave has a different icon though, which isn't handled in the delete.
We should either simplify the now redundant logic for deleteTextView where it sets text for groups in spite of being set to invisible, or else consolidate the deleteTextView behaviour by switching both the text and icon and remove the leaveTextView.

Reply:
Done